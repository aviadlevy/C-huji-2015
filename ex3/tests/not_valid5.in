4
0 0 0 0
0 0 1c 0
0 0 0 0
0 0 0 0
