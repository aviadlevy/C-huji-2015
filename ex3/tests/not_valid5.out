./tests/not_valid5.in: not a valid suduku file
